src/imap.rs

```diff
@@ -87,7 +88,7 @@ pub(crate) struct Imap {
 
     oauth2: bool,
 
-    login_failed_once: bool,
+    pub(crate) login_failed_once: Option<bool>,
```

Comment:
readability(blocking): It took me a while to understand what the `Option<bool>` is doing here.

Instead, `connect(..)` could just get a parameter `notify_about_wrong_pw` or `configuring` or similar that is checked before notifying. Since `connect(..)` only has two call sites, the change will be minimal.

Reply:
Adding `configuring` to `connect(&mut self, context: &Context)` looks more or less ok, but definitely not `notify_about_wrong_pw`, it's too particular thing unrelated to `Imap` itself. Will try to add `configuring` then